docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -784,7 +784,12 @@ for the data files and partition the storage per day using the column
 Updating the data in the materialized view with
 :doc:`/sql/refresh-materialized-view` deletes the data from the storage table,
 and inserts the data that is the result of executing the materialized view
-query into the existing table.
+query into the existing table. Refreshing the materialized view will also store
+the snapshot-ids of all the tables that are part of the materialized
+view's query at that point. When the materialized view is queried, the snapshot-ids
+are used to check if the data in the storage table is up to date. If the data is
+outdated, the storage is considered stale and the materialized view will behave 
```

Comment:
`If the data in the storage table is outdated, the materialized view will behave..`